decred/decred/wallet/simple.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright (c) 2019, Brian Stafford
+Copyright (c) 2019, The Decred developers
+See LICENSE for details
+"""
+
+import os
+
+from decred import DecredError
+from decred.crypto import crypto, mnemonic, rando
+from decred.dcr import nets
+from decred.dcr.dcrdata import DcrdataBlockchain
+from decred.util import chains, database
+from decred.util.helpers import mkdir
+from decred.wallet.wallet import Wallet
+
+
+class DefaultSignals:
+    """DefaultSignals prints the balance to stdout."""
+
+    @staticmethod
+    def balance(b):
+        print(repr(b))
+
+
+def paths(base, network):
+    """
+    Get the paths for the network directory, the wallet database file, and the
+    blockchain database file.
+    """
+    netDir = os.path.join(base, network)
+    dbPath = os.path.join(netDir, "wallet.db")
+    dcrPath = os.path.join(netDir, "dcr.db")
+    return netDir, dbPath, dcrPath
+
+
+DCRDATA_PATHS = {
+    "mainnet": "https://explorer.dcrdata.org/",
+    "testnet3": "https://testnet.dcrdata.org/",
+    "simnet": "http://localhost:17779",
+}
+
+
+class SimpleWallet(Wallet):
+    """
+    SimpleWallet is a single-account Decred wallet.
+    """
+
+    def __init__(self, walletDir, pw, network, signals=None, allowCreate=False):
+        """
+        Args:
+            dir (str): A directory for wallet database files.
+            pw (str): The user password.
+            network (str): The network name.
+            signals (Signals): A signal processor.
+        """
+        signals = signals if signals else DefaultSignals
+        self.net = net = nets.parse(network)
```

Comment:
Can we call net netParams for clarity? It looks like we are adopting that pattern in other places.